### PR TITLE
feat(initial-datasource): always select `grafanacloud-logs` as the default datasource

### PR DESCRIPTION
**Pull request by svennergr**
With this PR the default datasource will be the datasource with the `grafanacloud-logs` uid. This `uid` is set for the stack's default logs datasource.

### CODE REVIEW

**Review by cyriltovena — APPROVED** [review]
LGTM

File: src/Components/IndexScene/IndexScene.tsx [hunk 1]
```diff
@@ -62,7 +61,8 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
     super({
       $timeRange: state.$timeRange ?? new SceneTimeRange({}),
       $variables:
-        state.$variables ?? getVariableSet(state.initialDS ?? getLastUsedDataSourceFromStorage(), state.initialFilters),
+        state.$variables ??
+        getVariableSet('grafanacloud-logs' ?? getLastUsedDataSourceFromStorage(), state.initialFilters),
```
Comment (matyax) [inline src/Components/IndexScene/IndexScene.tsx:65]: What would happen if, for whatever reason, there's no `grafanacloud-logs` in the instance? I think the first use could be one example, where you don't have either `grafanacloud-logs` and last used from storage.

Comment (svennergr) [reply src/Components/IndexScene/IndexScene.tsx:65]: It just selects the first from the list.

image
-> 
image

Comment (matyax) [reply src/Components/IndexScene/IndexScene.tsx:65]: Fantastic!

**Pull request merged**